project/templates/edit_inventory.html

```diff
@@ -3,6 +3,28 @@
 {% block title %}Pantry Inventory{% endblock %}
 
 {% block content %}
+    <script>
+        function updateForm() {
```

Comment:
Oh some JS? Are we allowed to code in that language?

Reply:
I used this so we can have dynamically created forms when editing units and expiration